docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -284,6 +284,16 @@ web-0
 web-1
 ```
 
+Note, if you instead see 403 Forbidden responses for the above curl command,
+you will need to fix the permissions of the directory mounted by the `volumeMounts`
+(due to a [bug when using hostPath volumes](https://github.com/kubernetes/kubernetes/issues/2630)) with:
+
+```shell
+for i in 0 1; do kubectl exec web-$i -- sh -c 'chmod 755 /usr/share/nginx/html'; done
```

Comment:
Thanks @enisoc , I removed the sh as suggested.

Yes, the perms on index.html are fine:

```
michael@hpmor2:~/dev/k8s/statefulsets$ for i in 0 1; do kubectl exec web-$i -- ls -la /usr/share/nginx/html/; done
total 8
drwxr-xr-x 2 root root   60 Jan 25 09:23 .
drwxr-xr-x 4 root root 4096 Jan 16 06:20 ..
-rw-r--r-- 1 root root    6 Jan 25 09:23 index.html
total 8
drwxr-xr-x 2 root root   60 Jan 25 09:23 .
drwxr-xr-x 4 root root 4096 Jan 16 06:20 ..
-rw-r--r-- 1 root root    6 Jan 25 09:23 index.html
```